hack/e2e/run.sh

```diff
@@ -155,9 +149,9 @@ if [[ "${UP}" = "yes" ]]; then
 fi
 
 pushd ./tests/e2e
-${GINKGO_BIN} . -p -nodes="${GINKGO_NODES}" -v --focus="${GINKGO_FOCUS}" --skip="${GINKGO_SKIP}" "" -- -kubeconfig="${KUBECONFIG}" -report-dir="${test_run}" -gce-zone="${ZONES%,*}" "${EXPANDED_TEST_EXTRA_FLAGS}"
+ginkgo . -p -nodes="${GINKGO_NODES}" -v --focus="${GINKGO_FOCUS}" --skip="${GINKGO_SKIP}" "" -- -kubeconfig="${KUBECONFIG}" -report-dir="${test_run}" -gce-zone="${ZONES%,*}" "${EXPANDED_TEST_EXTRA_FLAGS}"
 popd
 
 if [[ "${DOWN}" = "yes" ]]; then
-    kops delete cluster --name "${CLUSTER_NAME}" --yes
+    ${test_run}/kops delete cluster --name "${CLUSTER_NAME}" --yes
```

Comment:
how did the binary get here? is kops also an e2e-tool?

Reply:
Good question, its downloaded by the kubetest2 kops deployer: https://github.com/kubernetes/kops/blob/master/tests/e2e/pkg/kops/download.go#L32-L55

Reply:
should we call kubetest2 --down (or whatever the right arguments are) then instead, so we don't have to care about that implementation detail?